dags/blocksec_plugin/rex_bank_deposit.py

```diff
@@ -1,55 +1,17 @@
-from airflow.models.baseoperator import BaseOperator
 from airflow.utils.decorators import apply_defaults
-from blocksec_plugin.web3_hook import Web3Hook
-from blocksec_plugin.ethereum_wallet_hook import EthereumWalletHook
+from blocksec_plugin.abis import VAULT_DEPOSIT_ABI
```

Comment:
Lets make this `REX_BANK_ABI`, the contract address for the bank is: 0xaD39F774A75C7673eE0c8Ca2A7b88454580D7F53